float32 goal_position
---
bool goal_accepted